runtime_lib/test_lib/CMakeLists.txt

```diff
@@ -16,6 +16,15 @@ target_include_directories(test_lib PRIVATE
     ${LibXAIE_INC_DIR}
 )
 
+# Linking against AIR so we can get access to the runtime functionality
+#if (DEFINED LibXAIE_${target}_DIR)
+
+if (${AIE_RUNTIME_TARGET} STREQUAL "x86_64")
+  message("Linking against AIR runtime at ${MLIR_AIR_DIR}")
+  target_link_libraries(test_lib "${MLIR_AIR_DIR}/build-pcie/runtime_lib/x86_64/airhost/libairhost.a")
+  target_include_directories(test_lib PRIVATE "${MLIR_AIR_DIR}/build-pcie/runtime_lib/x86_64/airhost/include/")
+endif()
+
```

Comment:
Matching on x86_64 doesn't work. We have x86 targets that are not pcie.

Reply:
Will add a summary below, but created a new target `x86_64-hsa` which allows us to differentiate between the multiple x86_64 backends.